source/dotnet/Library/AdaptiveCards/AdaptiveCardConverter.cs

```diff
@@ -44,5 +48,30 @@ public override bool CanConvert(Type objectType)
         {
             return typeof(AdaptiveCard).GetTypeInfo().IsAssignableFrom(objectType.GetTypeInfo());
         }
+
+        private AdaptiveCard MakeFallbackTextCard(JObject jObject)
+        {
+            // Retrieve fallbackText and language defined by parsed json
+            string language = jObject.Value<string>("language");
+            string fallbackText = jObject.Value<string>("fallbackText");
+
+            if (fallbackText == null || fallbackText == "")
+                fallbackText = "We're sorry, this card couldn't be displayed";
+
+            // Define AdaptiveCard to return
+            AdaptiveCard fallbackTextCard = new AdaptiveCard("1.0");
+            fallbackTextCard.BackgroundImage = null;
+            fallbackTextCard.Lang = language ?? "en";
+            fallbackTextCard.Speak = "";
+            fallbackTextCard.Body.Add(new AdaptiveTextBlock
+            {
+                Text = fallbackText
+            });
+
+            // Add relevant warning
+            Warnings.Add(new AdaptiveWarning(7, "Schema version is not supported"));
```

Comment:
why `7`?

Reply:
We should have a common set of error codes that are shared amongst the platform. But that's beyond this scope. I would define the error codes in an enum and use those values so at least it's explicit and consolidated.